packages/modules/src/utils/Types.ts

```diff
@@ -47,31 +59,42 @@ export enum StorageType {
 }
 
 export type SessionParams = {
+  /** Redundant now as we've favoured uuid() */
   sessionID?: string;
+  /** Session Signer: viemWallet or ethers signer. Ingested when passed into smartAccount */
   sessionSigner: SupportedSigner;
+  /** The session validation module session validation module is a sub-module smart-contract which works with session key manager validation module. It validates the userop calldata againsnt the defined session permissions (session key data) within the contract. */
   sessionValidationModule?: Hex;
+  /** Additional info if needed to be appended in signature */
   additionalSessionData?: string;
 };
 
 export type ModuleInfo = {
   // Could be a full object of below params and that way it can be an array too!
   // sessionParams?: SessionParams[] // where SessionParams is below four
   sessionID?: string;
+  /** Session Signer: viemWallet or ethers signer. Ingested when passed into smartAccount */
   sessionSigner?: SupportedSigner;
+  /** The session validation module session validation module is a sub-module smart-contract which works with session key manager validation module. It validates the userop calldata againsnt the defined session permissions (session key data) within the contract. */
```

Comment:
Just some small typos here:
- "againsnt"
- "The session validation module session validation module"

Reply:
Thanks 👍